docs/netobserv_cli.adoc

```diff
@@ -0,0 +1,69 @@
+// Automatically generated by './scripts/generate-doc.sh'. Do not edit, or make the NETOBSERV team aware of the editions.
+:_mod-docs-content-type: REFERENCE
+[id="network-observability-cli-usage_{context}"]
+= Network Observability CLI usage
+
+Users of Netobserv CLI can pass command line arguments to enable features or pass configuration options to the eBPF agent and flowlogs-pipeline.
+This document lists all the supported command line options and their possible values.
+
+== Flows usage
+[cols="1,1,1",options="header"]
+|===
+| Option | Description | Default
+|--enable_pktdrop|  enable packet drop                  | false
+|--enable_dns|      enable DNS tracking                 | false
+|--enable_rtt|      enable RTT tracking                 | false
+|--enable_filter|   enable flow filter                  | false
+|--log-level|       components logs                     | info
+|--max-time|        maximum capture time                | 5m
+|--max-bytes|       maximum capture bytes               | 50000000 = 50MB
+|--copy|            copy the output files locally       | prompt
+|--direction|       flow filter direction               | n/a
+|--cidr|            flow filter CIDR                    | 0.0.0.0/0
+|--protocol|        flow filter protocol                | n/a
+|--sport|           flow filter source port             | n/a
+|--dport|           flow filter destination port        | n/a
+|--port|            flow filter port                    | n/a
+|--sport_range|     flow filter source port range       | n/a
+|--dport_range|     flow filter destination port range  | n/a
+|--port_range|      flow filter port range              | n/a
+|--tcp_flags|       flow filter TCP flags               | n/a
+|--icmp_type|       ICMP type                           | n/a
+|--icmp_code|       ICMP code                           | n/a
+|--peer_ip|         peer IP                             | n/a
+|--action|          flow filter action                  | Accept
+|--interfaces|      interfaces to monitor               | n/a
+|===
+
+Example running flow capture with some options:
+```
+oc netobserv flows --enable_pktdrop=true  --enable_rtt=true --enable_filter=true --action=Accept --cidr=0.0.0.0/0 --protocol=TCP --port=49051
+```
+== Packets usage
```

Comment:
```suggestion
== Packets capture options
You can filter on port and protocol for packet capture data. 

.`oc netobserv packets` syntax
[source,terminal]
----
$ oc netobserv packets [<option>]
----
```

Reply:
I would like to retain some intro text for the table. I guess packet capture is more beefy in this release so its more than just port and protocol filtering.